extensions/import/src/wizard/pages/fileConfigPage.ts

```diff
@@ -340,7 +340,14 @@ export class FileConfigPage extends ImportPage {
 		let connectionUri = await sqlops.connection.getUriForConnection(this.model.server.connectionId);
 		let queryProvider = sqlops.dataprotocol.getProvider<sqlops.QueryProvider>(this.model.server.providerName, sqlops.DataProviderType.QueryProvider);
 
-		let query = `SELECT name FROM sys.schemas`;
+		let query = '';
+		if (this.databaseDropdown.value != undefined)
+		{
+			query = `SELECT name FROM [${this.databaseDropdown.value}].sys.schemas`;
```

Comment:
This also has a sql injection vulnerability for database names containing a closing square bracket. One of the tasks we put on our backlog when creating this extension was to create a secure way of running queries that accept external input like this but unfortunately we haven't done that work yet and I think this fix will need to wait on it

Reply:
Please explain to me how you expect SQL injection? The first list is generated and placed into a drop down from the server connection. You can only select a database from the list. The section list is generated from the database which is selected and placed into another list. You cannot get SQL injection that way. You are forcing the parameters. Even so, if you are worried about the drop downs being compromised, we could easily add a 'safe' string and remove any foreign characters to clean this up.

Reply:
@aspnerd I think the concern is if some creates a database name that contains some type of escape sequence that this could be a problem.  Perhaps it would only mess up the UI rendering, but not actually be security issue.

@MattIrv do we have any concrete issues with this PR?  Or suggestions on how to make it ready for check-in?  I'd like to either merge or close out some of the older PRs.

Reply:
@kburtram if your database name has any escape character in it's name, you probably have more serious issues to attend to. That is why I wrapped it in brackets. It shouldn't mess up the UI that way I believe.

Reply:
It may be safe if we remove any square bracket characters from the database name before sending it to the server, though it would block certain database names from working here